pyresample/test/test_geometry.py

```diff
@@ -1276,6 +1298,55 @@ def test_area_def_geocentric_resolution(self):
         geo_res = area_def.geocentric_resolution()
         np.testing.assert_allclose(248.594116, geo_res)
 
+    @unittest.skipIf(CRS is None, "pyproj 2.0+ required")
+    def test_area_def_init_projection(self):
+        """Test AreaDefinition with different projection definitions."""
+        proj_dict = {
+            'a': '6378144.0',
+            'b': '6356759.0',
+            'lat_0': '90.00',
+            'lat_ts': '50.00',
+            'lon_0': '8.00',
+            'proj': 'stere'
+        }
+        crs = CRS(CRS.from_dict(proj_dict).to_wkt())
```

Comment:
Just curious: What is the benefit on the extra `CRS(crs.to_wkt())` layer here?

Reply:
It is needed otherwise the CRS objects are not strictly equal later on. The conversion is done inside the AreaDefinition so when I do `area_def.crs == crs` they are the same. If I remember correctly if I don't do it then there is some information gained or lost in the conversion to WKT that makes the CRS objects not equal.

Reply:
Just checked and the WKT of the two CRS objects is the same, but something in the CRS based on the PROJ dict is not equivalent.

Reply:
Oh boy! Thanks for sorting out all these nasty details!